docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -65,21 +65,9 @@ dotnet_naming_symbols.types.applicable_kinds = class, struct, interface, enum, d
 dotnet_naming_symbols.types.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
 ```
 
-## Naming rule properties
-
-All naming rule properties are required for a rule to take effect.
-
-| Property | Description |
-| -- | -- |
-| `symbols` | The title of a symbol group; the naming rule will be applied to the symbols in this group |
-| `style` | The title of the naming style which should be associated with this rule |
-| `severity` |  Sets the severity with which to enforce the naming rule. Set the associated value to one of the available [severity levels](../configuration-options.md#severity-level).<sup>1</sup> |
+## \<propertyName> and \<propertyValue>
```

Comment:
```suggestion
## \<propertyName> and \<propertyValue>

(Can you add some intro here?)
```

Reply:
I've done so in https://github.com/dotnet/docs/pull/28237/commits/379839b36f8802c7dc94ab5285240d25cca1c7be